scenes/src/app/scenes/badge/badge-scene.scss

```diff
@@ -0,0 +1,4 @@
+.scene-badge-icon {
+  color: palevioletred;
```

Comment:
oh I think this is why I was trippin about the colors, what if you use a dark gray instead? maybe something close to `#333`?